README.md

```diff
@@ -95,9 +98,10 @@ module.exports = {
     {
       resolve: `gatsby-mdx`,
       options: {
-        defaultLayout: require.resolve(
-          "./src/components/default-page-layout.js"
-        )
+        defaultLayouts: {
+          posts: require.resolve("./src/components/posts-layout.js"),
+          default: require.resolve("./src/components/default-page-layout.js")
```

Comment:
there's no `gatsby-source-filesystem` in the plugins list so it's a bit confusing what the keys relate to in this config.